docs/reference/commandline/dockerd.md

```diff
@@ -921,7 +921,7 @@ This is a full example of the allowed configuration options in the file:
 	"tlscacert": "",
 	"tlscert": "",
 	"tlskey": "",
-	"api-cors-headers": "",
+	"api-cors-header": "",
```

Comment:
Same for this one; we should probably add a fallback for the "wrong" one; https://github.com/docker/docker/blob/e9117578a7837eda31d690d5e51d09d43a6a8fca/daemon/config.go#L91